app/js/reducers/draftLabOrderReducer.js

```diff
@@ -82,11 +82,28 @@ export default (state = initialState.draftLabOrderReducer, action) => {
       const { orderId, orderUrgency } = action.order;
       return {
         ...state,
-        draftLabOrders: state.draftLabOrders.map(draftOrder => (draftOrder.id === orderId ?
-          { ...draftOrder, urgency: orderUrgency }
-          : draftOrder)),
+        draftLabOrders: state.draftLabOrders.map((draftOrder) => {
+          let order;
+          if (draftOrder.id === orderId) {
+            if (draftOrder.tests) {
+              order = {
+                ...draftOrder,
+                urgency: orderUrgency,
+                tests: draftOrder.tests.map(_test => ({
+                  ..._test, urgency: orderUrgency,
```

Comment:
why did you decide to use a dangling-underscore with your parameter naming

Reply:
Fixed :-)